torch/_export/_experimental.py

```diff
@@ -0,0 +1,42 @@
+from typing import List, Any, Dict, Optional
+
+import torch
+import torch.fx._pytree as fx_pytree
+from torch._export.exported_program import ExportedProgram
+from torch._inductor.compile_fx import compile_fx_aot
+
+__all__ = ["aot_compile"]
+
+def aot_compile(
```

Comment:
We can't have torch._inductor.aot_compile take in an ExportedProgram because of circular import errors...
But another solution is that because we're in python we could also just ignore the annotations and pass torch._inductor.aot_compile an ExportedProgram and import + check inside that function.